.gitignore

```diff
@@ -12,3 +12,5 @@ compiled
 ethers
 site
 ethers-staking
+*.heapprofile
+*.cpuprofile
```

Comment:
Are these files being created by `runner.js` I didn't notice that this was the case.

Reply:
They are created by Node.js profiler. We don't need them now. I will remove them